cmd/node/config/config.toml

```diff
@@ -486,6 +486,10 @@
     MaxRoundsToKeepUnprocessedMiniBlocks = 300   # max number of rounds unprocessed miniblocks are kept in pool
     MaxRoundsToKeepUnprocessedTransactions = 300 # max number of rounds unprocessed transactions are kept in pool
 
+[KeysHolderConfig]
+    IsMainMachine = false
```

Comment:
why do we have a separate configuration? Couldn't this be determined based on the redundancy level?

Reply:
we have this in prefs.toml file

Reply:
and the MaxRoundsWithoutReceivedMessages is actually the RedundancyLevel value